draft-ietf-quic-recovery.md

```diff
@@ -726,10 +700,9 @@ Pseudocode for OnAckReceived and UpdateRtt follow:
       // RTO was spurious. Otherwise, inform congestion control.
       if (rto_count > 0 &&
             smallest_newly_acked > largest_sent_before_rto):
```

Comment:
largest_sent_before_rto is still used here

Reply:
As well as `rto_count`